README.md

```diff
@@ -1,8 +1,42 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Dayde Reid - Jarvis Consulting
 
+I am a recent graduate of the University of Toronto Scarborough, where I earned my Honours Bachelor of Science. I studied computer science as a specialist program, focusing on software engineering, as I prefer the more practical side of computer science. During my schooling, I managed to get a bit of experience in the workplace, as I had two co-op job positions. The first was an 8-month term at Parks Canada, where I helped developed and release a guide app for one of their parks for both iOS and Android. The second was at Loblaw Digital, where I spent 4 months improving and maintaining one of there existing ecommerce websites. I joined the Jarvis Consulting Group a few months after graduating and began training for a Data Engineering position. Right now, I am looking for a position at a company where I can put the skills I have learned to good use, while hopefully also continuing to learn so that I might better myself further.
+
+## Skills
+
+**Proficient:** Java, Javascript, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Python, C, C++, React
+
+**Familiar:** Swift, Angular, Matlab, Haskell
+
+## Development Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_DaydeReid](https://github.com/jarviscanada/jarvis_data_eng_DaydeReid)
+
+- **[Cluster Monitor](./linux_sql)**: Implemented a series of bash scripts that record hardware specifications and usage statistics for a Linux cluster and write them to a PSQL database contained in a Docker instance for future analysis
+- **[Core Java Apps](./core_java)**: Developed three small Java apps: An app that replicates the recursive grep function found in Linux systems using Java streams, some basic DAO implementations for a JDBC system, and a Twitter CLI App, which utilizes a REST API to post, view, and delete tweets from the Twitter webpage 
```

Comment:
* talk about how you manage your project using maven
* talk how you tested your applications